doc/topics/tutorials/gitfs.rst

```diff
@@ -796,6 +791,113 @@ To map a branch other than ``master`` as the ``base`` environment, use the
 The base can also be configured on a :ref:`per-remote basis
 <gitfs-per-remote-config>`.
 
+Use Case: Code Promotion (dev -> qa -> base)
+--------------------------------------------
+
+When running a :ref:`highstate <running-highstate>`, the ``top.sls`` files from
+different from all of the branches and tags will be merged into one. This does
```

Comment:
This wording seems a bit off. "from different from"

Reply:
Yeah, not what I originally wanted to type here, this is fixed.